Deferred/Deferred.swift

```diff
@@ -146,11 +224,26 @@ extension Deferred {
 }
 
 extension Deferred {
+    /**
+    Composes the recieving value with another.
```

Comment:
Ah, that is nice. I'm used to Obj-C-derived types where you can always just throw them all in the `AnyObject` bucket, but that is an issue for actual Swift code.